lib/dal/src/management/prototype.rs

```diff
@@ -108,6 +119,31 @@ pub struct ManagedComponent {
     geometry: HashMap<String, ManagementGeometry>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct ManagerComponent {
+    #[serde(flatten)]
+    component: ManagedComponent,
+    incoming_connections: HashMap<String, SocketRefsAndValues>,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
incoming connections are passed as a singleton if the socket arity is "one", and an array if the arity is "many". Even if there is only one connection, sockets with arity="many" get an array with one value, for consistency.